test/karma/bool.cpp

```diff
@@ -117,7 +117,7 @@ int main()
 
         bool b = false;
         BOOST_TEST(test("false", bool_, phoenix::ref(b)));
-        BOOST_TEST(test("true", bool_, ++phoenix::ref(b)));
+        BOOST_TEST(test("true", bool_, !phoenix::ref(b)));
```

Comment:
`warning: incrementing expression of type bool is deprecated and incompatible with C++1z [-Wdeprecated-increment-bool]`